CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.8)
+project(penguinV)
+
+option(PENGUINV_BUILD_TEST "Build tests of penguinV" ON)
+if(${PENGUINV_BUILD_TEST} OR (CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR))
+    enable_testing()
+    add_subdirectory(test)
+endif()
```

Comment:
A small note: each file should be ended with an empty line at the bottom. Check red circle and arrow at this line :)

Reply:
Now I know what those red circles means ;)